acceptancetests/jujupy/client.py

```diff
@@ -2121,6 +2128,90 @@ def switch(self, model=None, controller=None):
             raise ValueError('No target to switch to has been given.')
         self.juju('switch', (':'.join(args),), include_e=False)
 
+    def scp(self, _from, _to):
+        self.juju('scp', (_from, _to))
+
+
+# caas `add-k8s` did not implement parsing kube config path via flag,
+# so parse it via env var ->  https://github.com/kubernetes/client-go/blob/master/tools/clientcmd/loader.go#L44
+KUBE_CONFIG_PATH_ENV_VAR = 'KUBECONFIG'
+
+
+class CaasClient(ModelClient):
+
+    cloud_name = 'k8cloud'
+    kubectl_path = None
+    kube_home = None
+    kube_config_path = None
+
+    def _ensure_dir(self, path):
+        try:
+            os.makedirs(path)
+        except OSError:
+            if not os.path.isdir(path):
+                raise
+
+    def __prepare(self):
```

Comment:
This should just be in the ```__init__```.

Reply:
it was initially in `__init__` but it didnot work. because `juju_home` is `JUJU_HOME=/tmp/test-run` in contructor `__init__`, but in `context_manager`, `juju_home` was switched to `juju_home/juju-homes/<current-build>` which is what I need